pandas/tests/plotting/common.py

```diff
@@ -510,20 +493,6 @@ def is_grid_on():
                 obj.plot(kind=kind, grid=True, **kws)
                 assert is_grid_on()
 
-    def _maybe_unpack_cycler(self, rcParams, field='color'):
-        """
-        Compat layer for MPL 1.5 change to color cycle
```

Comment:
@WillAyd This sounded like it was only for compat. But I'll add it back in as convenience instead of the list comps